client/src/app/core/core-services/autoupdate.service.ts

```diff
@@ -137,15 +159,18 @@ export class AutoupdateService {
                     }
                 }
             }
+            fullListUpdate[collection] =
+                this.getDescriptorForCollection(originalRequest, collection)?.isFullList && isFirstResponse;
```

Comment:
This is now calculated for every update. Calculate it once when creating the AU-request and just pass the list of collections. Also: Switch around the `&&` since in most cases `isFirstResponse` will be false, so the check is faster.